{"level":"error","message":"Erro ao buscar a loja com ID 67792c4b08eb0f4: ðŸ˜¿ðŸ˜¿","stack":"CastError: Cast to ObjectId failed for value \"67792c4b08eb0f4\" (type string) at path \"_id\" for model \"Loja\"\n    at SchemaObjectId.cast (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\schema\\objectId.js:250:11)\n    at SchemaType.applySetters (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\schemaType.js:1255:12)\n    at SchemaType.castForQuery (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\schemaType.js:1673:17)\n    at cast (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\cast.js:390:32)\n    at Query.cast (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\query.js:4889:12)\n    at Query._castConditions (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\query.js:2306:10)\n    at model.Query._findOne (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\query.js:2630:8)\n    at model.Query.exec (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\query.js:4438:80)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async LojaService.findById (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\models\\lojas\\service\\lojaService.ts:77:26)"}
{"level":"error","message":"Erro ao buscar a loja com ID fgvyufyui: ðŸ˜¿ðŸ˜¿","stack":"CastError: Cast to ObjectId failed for value \"fgvyufyui\" (type string) at path \"_id\" for model \"Loja\"\n    at SchemaObjectId.cast (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\schema\\objectId.js:250:11)\n    at SchemaType.applySetters (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\schemaType.js:1255:12)\n    at SchemaType.castForQuery (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\schemaType.js:1673:17)\n    at cast (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\cast.js:390:32)\n    at Query.cast (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\query.js:4889:12)\n    at Query._castConditions (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\query.js:2306:10)\n    at model.Query._findOne (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\query.js:2630:8)\n    at model.Query.exec (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\mongoose\\lib\\query.js:4438:80)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async LojaService.findById (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\models\\lojas\\service\\lojaService.ts:77:26)"}
{"level":"error","message":"Erro ao buscar a loja com ID fgvyufyui: ðŸ˜¿ðŸ˜¿"}
{"level":"error","message":"Erro ao buscar a loja com ID SP: ðŸ˜¿ðŸ˜¿"}
{"level":"error","message":"Erro ao buscar a loja com ID SP: ðŸ˜¿ðŸ˜¿"}
{"level":"error","message":"Erro ao buscar a loja com ID SP: ðŸ˜¿ðŸ˜¿"}
{"level":"error","message":"Erro ao buscar loja pelo CEP 01000-000: ðŸ˜¿"}
{"level":"error","message":"Erro ao buscar loja pelo CEP 49037-050: ðŸ˜¿"}
{"level":"error","message":"Erro ao buscar informaÃ§Ãµes de frete"}
{"level":"error","message":"Erro ao buscar lojas por CEP:"}
{"level":"error","message":"Erro ao buscar informaÃ§Ãµes de frete"}
{"level":"error","message":"Erro ao buscar informaÃ§Ãµes de frete"}
{"level":"error","message":"Erro ao buscar lojas por CEP:"}
{"level":"error","message":"Erro ao buscar informaÃ§Ãµes de frete"}
{"level":"error","message":"Erro ao buscar informaÃ§Ãµes de frete"}
{"level":"error","message":"Erro ao buscar frete para a loja MEOWBLE 01:"}
{"level":"error","message":"Erro ao buscar informaÃ§Ãµes de frete"}
{"level":"error","message":"Erro ao buscar frete para a loja MEOWBLE 01:"}
{"level":"error","message":"Erro ao buscar informaÃ§Ãµes de frete"}
{"level":"error","message":"Erro ao buscar frete para a loja MEOWBLE 01:"}
{"level":"error","message":"Erro ao buscar informaÃ§Ãµes de frete"}
{"level":"error","message":"Erro ao buscar frete para a loja MEOWBLE 01:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro ao buscar lojas por CEP:"}
{"level":"error","message":"Erro ao buscar lojas por CEP:"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","mensagem":"Cannot read properties of undefined (reading 'valueOf')","message":"Erro ao buscar lojas por CEP: 05202-140","stack":"TypeError: Cannot read properties of undefined (reading 'valueOf')\n    at lojasMenor50Km.reduce.pdvs (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\lojas\\service\\findByCep\\categorizeStores.ts:12:44)\n    at Array.reduce (<anonymous>)\n    at categorizeStores (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\lojas\\service\\findByCep\\categorizeStores.ts:8:50)\n    at findByCepServiceService.findByCep (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\lojas\\service\\findByCep\\findByCepService.ts:70:59)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async LojaController.findByCep (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\lojas\\controller\\lojaController.ts:121:31)\n    at async C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro ao buscar lojas por CEP: 05202-140"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"level":"error","message":"Erro na chamada Ã  API dos Correios:"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 11:34:55"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 11:37:58"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 11:41:40"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 11:44:12"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 11:46:06"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 11:47:47"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 11:51:20"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 11:53:41"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 11:54:31"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 11:55:37"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 11:57:09"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 12:22:35"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 12:23:45"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 12:24:43"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 12:25:55"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 12:28:20"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 12:28:41"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 12:30:02"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 12:32:13"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 12:33:09"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 12:34:14"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 12:36:37"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:71:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 12:41:31"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:78:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 13:03:30"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:82:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 13:07:43"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:95:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 13:09:50"}
{"detalhes":"Error","level":"error","message":"Erro na chamada Ã  API dos Correios:: Erro na chamada Ã  API","stack":"Error: Erro na chamada Ã  API\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\correios\\test\\correioService.test.ts:95:13)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 13:11:02"}
{"detalhes":"Error","level":"error","message":"Erro ao validar o CEP: 12345-678: Network error","stack":"Error: Network error\n    at Object.<anonymous> (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\src\\validators\\test\\cepValidator.test.ts:43:43)\n    at Promise.then.completed (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:298:28)\n    at new Promise (<anonymous>)\n    at callAsyncCircusFn (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\utils.js:231:10)\n    at _callCircusTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:316:40)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at _runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:252:3)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:126:9)\n    at _runTestsForDescribeBlock (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:121:9)\n    at run (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\run.js:71:3)\n    at runAndTransformResultsToJestFormat (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapterInit.js:122:21)\n    at jestAdapter (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:19)\n    at runTestInternal (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:367:16)\n    at runTest (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\runTest.js:444:34)\n    at Object.worker (C:\\Users\\Catnip\\Documents\\DEV\\Estagio\\Meowble-1\\project\\node_modules\\jest-runner\\build\\testWorker.js:106:12)","timestamp":"2025-01-08 13:16:08"}
